types/jasmine-ajax/index.d.ts

```diff
@@ -5,14 +5,10 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-interface JasmineAjaxResponse {
-    status?: number;
-    statusText?: string;
-    responseText?: string;
-    response?: string;
-    contentType?: string;
-    responseHeaders?: { [key: string]: string };
-}
+/**
+ * @deprecated Use JasmineAjaxRequestStubReturnOptions instead
+ */
+type JasmineAjaxResponse = JasmineAjaxRequestStubReturnOptions;
```

Comment:
If the answer to my previous question was "yes", then I'd much prefer that you deprecate `JasmineAjaxRequestStubReturnOptions` rather than `JasmineAjaxResponse`.

Reply:
I still prefer this, simply because the latter name is shorter and the former is too specific (the type is not only for `stubRequest.andReturn` but also for `respondWith`.

Reply:
Ok, I'll make this solution:

> I'd much prefer that you deprecate `JasmineAjaxRequestStubReturnOptions` rather than `JasmineAjaxResponse`.

Reply:
Thanks!